flexmeasures/data/schemas/generic_assets.py

```diff
@@ -109,23 +156,16 @@ def validate_account(self, account_id: int | None):
             )
 
     @validates("attributes")
-    def validate_attributes(self, attributes: dict):
+    def validate_attributes(self, attributes: dict):  # noqa C901
         sensors_to_show = attributes.get("sensors_to_show", [])
 
-        # Check type
-        if not isinstance(sensors_to_show, list):
-            raise ValidationError("sensors_to_show should be a list.")
-        for sensor_listing in sensors_to_show:
-            if not isinstance(sensor_listing, (int, list)):
-                raise ValidationError(
-                    "sensors_to_show should only contain sensor IDs (integers) or lists thereof."
-                )
-            if isinstance(sensor_listing, list):
-                for sensor_id in sensor_listing:
-                    if not isinstance(sensor_id, int):
-                        raise ValidationError(
-                            "sensors_to_show should only contain sensor IDs (integers) or lists thereof."
-                        )
+        # Use SensorsToShowSchema to validate sensors_to_show
+        try:
+            SensorsToShowSchema()._deserialize(
```

Comment:
Maybe only if `sensors_to_show` is not ` None`?

Reply:
done